docs/products/dragonfly/howto/eviction-policy-df.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Data eviction policy in Aiven for Dragonfly
+sidebar_label: Data eviction policy
+---
+
+Aiven for Dragonfly® optimizes cache memory management with a data eviction policy, ensuring efficient memory usage and optimal performance.
```

Comment:
```suggestion
Aiven for Dragonfly® optimizes cache memory management with a low-overhead data eviction policy.
```
sounds better imo. Original sounds like we're trying too hard to oversell it. But up to you.

Reply:
Updated to the suggestion.